cumulusci/tasks/salesforce/Deploy.py

```diff
@@ -69,7 +69,14 @@ def _init_options(self, kwargs):
 
     def _get_api(self, path=None):
         if not path:
-            path = self.task_config.options__path
+            path = self.options.get("path")
+
+            if not path:
+                # Hopefully this is never triggered and we can delete it.
+                self.logger.warn(
+                    "Getting self.task_config.options__path in _get_api is deprecated"
+                )
+                path = self.task_config.options__path
```

Comment:
I don't think this is needed. If "path" was set in the task_config's options, then it will also be there in self.options.

Reply:
Do you have any guess why we looked it up in self.task_config.options__path ? Just a misunderstanding of how the framework works? I wasn't sure if there was a subtlety that I missed.

Reply:
I dug around in git blame and it looks like it was just written a long time ago when self.options was always a reference to self.task_options and there was no _init_options method.

Reply:
So not so much a misunderstanding as that the difference between the two didn't matter.